loopout.go

```diff
@@ -52,11 +52,19 @@ const (
 	DefaultHtlcConfTarget = 6
 
 	// DefaultSweepConfTargetDelta is the delta of blocks from a Loop Out
-	// swap's expiration height at which we begin to use the default sweep
-	// confirmation target.
-	//
-	// TODO(wilmer): tune?
-	DefaultSweepConfTargetDelta = DefaultSweepConfTarget * 2
+	// swap's expiration height at which we begin to cap sweep confirmation
```

Comment:
nit: ... at which be begin to cap the sweep...

Reply:
Fixed. Thanks!